build/circle_tests.sh

```diff
@@ -3,4 +3,6 @@
 set -x
 set -e
 
+docker run -t --entrypoint="/usr/bin/run_unittests" -w "/root/src/preprocessing-workflow" oesteban/fmriprep
```

Comment:
BTW, if you want CircleCI to understand this test as a separate test (and run always the second line of this file), then you will need to put this line as a new entry in the ```circle.yml``` file. Actually, having this file is not necessary here, we could also get the smoke test out to the yml file and remove this build/circle_tests.sh